src/libostree/ostree-bootloader-zipl.c

```diff
@@ -67,6 +72,25 @@ _ostree_bootloader_zipl_get_name (OstreeBootloader *bootloader)
   return "zipl";
 }
 
+static gboolean
+_ostree_secure_execution_mount(GError **error)
+{
+  const char *device = realpath (SECURE_EXECUTION_PARTITION, NULL);
+  if (device == NULL)
+    return glnx_throw_errno_prefix(error, "s390x SE: resolving %s", SECURE_EXECUTION_PARTITION);
+  if (mount (device, SECURE_EXECUTION_MOUNTPOINT, "ext4", 0, NULL) < 0)
```

Comment:
I think this is OK as is, but one thing we could consider here is reworking the zipl invocation to be a systemd unit we activate.  In that case, we can start using all the systemd features, like device dependencies, handling mount namespaces etc.

Reply:
I'm thinking about moving all SecExe related stuff into zipl itself. Maybe a wrapper within `s390-tools`. 
Wrapper (without any bash scripts) exists now as part of proposed [PR](https://github.com/coreos/coreos-installer/pull/851) for coreos-installer